velox/functions/prestosql/aggregates/SetAggregates.cpp

```diff
@@ -199,8 +199,11 @@ class SetBaseAggregate : public exec::Aggregate {
 template <typename T>
 class SetAggAggregate : public SetBaseAggregate<T> {
  public:
+  SetAggAggregate(const TypePtr& resultType, const bool throwOnNestedNulls)
+      : SetBaseAggregate<T>(resultType),
+        throwOnNestedNulls_(throwOnNestedNulls) {}
   explicit SetAggAggregate(const TypePtr& resultType)
```

Comment:
add empty line

Do we need 2 constructors? Can we use one?

Reply:
Fixed